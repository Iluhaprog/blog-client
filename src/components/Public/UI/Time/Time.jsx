import styled from 'styled-components';

const Time = styled.time`
  max-width: fit-content;
  font-size: 14px;
  color: rgba(28, 28, 28, 0.5);
  display: flex;
  justify-content: space-between;
  align-items: center;
  
  &:after,
  &:before {
    display: block;
    content: '';
    margin: 0 5px;
    width: 3px;
    height: 3px;
    border-radius: 50%;
    background-color: rgba(28, 28, 28, 0.5);
  }
`;

export {Time};
