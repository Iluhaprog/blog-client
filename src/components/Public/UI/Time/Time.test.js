import {Time} from './Time';
import {render} from '@testing-library/react';

test('Should render Time component', () => {
  const {asFragment} = render(
    <Time>
      TEST_TIME
    </Time>
  );
  expect(asFragment()).toMatchSnapshot();
});
