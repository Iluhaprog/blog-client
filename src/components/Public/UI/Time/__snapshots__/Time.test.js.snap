// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Time component 1`] = `
<DocumentFragment>
  <time
    class="sc-bdfBwQ hWcxSz"
  >
    TEST_TIME
  </time>
</DocumentFragment>
`;
